oasis/abi

```diff
@@ -18,3 +18,4 @@ Library abi_plugin
   BuildDepends:     bap, bap-abi
   InternalModules:  Abi_main
   XMETADescription: apply abi information to a project
+  XMETAExtraLines:  tags="core, pass"
```

Comment:
maybe `basic` is a better name, as it provides the `basic` functionality needed by all frontends. However, you may leave the tags as they are, I'm leaving this comment for discussion only. We will rearrange the features names several time before the release. The main idea, is that the downstream users shouldn't notice, that we introduced this filter - thus all the existing plugins should be loaded automatically, thus should be in the same category.